kivy/lib/sdl2.pxi

```diff
@@ -586,6 +586,8 @@ cdef extern from "SDL.h":
     cdef SDL_bool SDL_HasScreenKeyboardSupport()
     cdef SDL_bool SDL_IsScreenKeyboardShown(SDL_Window *window)
     cdef void SDL_GL_GetDrawableSize(SDL_Window *window, int *w, int *h)
+    cdef int SDL_SetWindowOpacity(SDL_Window * window, float opacity)
```

Comment:
I'm not sure how this even compiles for you as the calls don't appear in the [SDL.h](https://hg.libsdl.org/SDL/file/77da466731bc/include/SDL.h) file (you use `cdef extern from "SDL.h":`)and that's probably why gcc doesn't like it ([implicit function declaration](https://travis-ci.org/kivy/kivy/jobs/231139054#L1439)). Try to pull the calls from [SDL_video.h](https://hg.libsdl.org/SDL/file/77da466731bc/include/SDL_video.h) instead.